PORTAL/jobs/_job_compile_bench.py

```diff
@@ -289,8 +319,12 @@ def build_pyperformance_config(req, bfiles):
     return cfg
 
 
-def build_compile_script(req, bfiles, fake=None):
-    fakedelay = FAKE_DELAY
+def build_compile_script(
+        req: CompileBenchRequest,
+        bfiles: _utils.FSTree,
+        fake: Optional[Any] = None
+) -> str:
+    fakedelay: Optional[int] = FAKE_DELAY
```

Comment:
Because it's set to an int on this line, but every branch of the `if` block below sets it to `None`, so you have to tell it that's ok to change the type.

Reply:
Ah, I looked at the old code and get it now.  Thanks for explaining.